Makefile

```diff
@@ -89,6 +89,8 @@ get-deps:
 	go get golang.org/x/tools/cover
 	go get golang.org/x/tools/cmd/cover
 	go get golang.org/x/tools/cmd/goimports
+	go get github.com/fzipp/gocyclo
```

Comment:
am pretty sure i added this https://github.com/aws/amazon-ecs-init/blob/38904632b2cb09c768b71321c7e9aaf64622740e/Makefile#L109 🤔